CHANGELOG.md

```diff
@@ -55,6 +55,8 @@
 - Fixed `onReadyForDisplay` not being called [#2721](https://github.com/react-native-video/react-native-video/pull/2721)
 - Fix type of `_eventDispatcher` on iOS target to match `bridge.eventDispatcher()` [#2720](https://github.com/react-native-video/react-native-video/pull/2720)
 
+- Fix crash when the source is a cameraroll [#2639] (https://github.com/react-native-video/react-native-video/pull/2639)
```

Comment:
I aprouve the PR, but can you double check the changelog change is in the good section ?

Reply:
done, I move it to Version 6.0.0-alpha.2